ActivityListener/.lambda-test-tool/SavedRequests/PersonUpdate.json

```diff
@@ -0,0 +1,20 @@
+{
+  "Records": [
```

Comment:
Why do we have 2 of the same files?

Reply:
These saved requests shouldn't be checked in - I will remove.